### PR TITLE
Create margin-top class to add more space above specified h3 headers.…

### PR DESCRIPTION
**Pull request by lmarkzon**
## Description

Issue #503 
Create and apply margin-top class to double space above the specified h3 headers below. So before they were 30px above the headers and now there is 60px.

**Typefaces**

**Pairings + Styles**

**Palette**

**Form Controls/Accessibility**

**Form Templates/Accessibility**

… Apply class to appropriate h3s.

### CODE REVIEW

**Comment by juliaelman** [comment]
Thanks for the pull request @lmarkzon! Would you mind adding a screenshot with an example of this solve?

**Comment by lmarkzon** [comment]
@juliaelman yes sorry about that - just added them!

**Comment by juliaelman** [comment]
@lmarkzon no apologies needed and thank you for your contribution! 😄

**Comment by ericadeahl** [comment]
👍 Looks good to go!

File: src/stylesheets/elements/_typography.scss [hunk 1]
```diff
@@ -45,6 +45,10 @@ h6 {
   margin-top: 1.5em;
 }
 
+h3.usa-heading.margin-top {
+  margin-top: 3em;
+}
```
Comment (maya) [inline src/stylesheets/elements/_typography.scss]: This is a styleguide-only change so should be in `docs/doc_assets/css/styleguide.scss`.

**Comment by maya** [comment]
@lmarkzon can you change the class to `heading-margin-alt` and use it as the single selector in the SCSS (i.e. not chained or nested to another selector)?

**Comment by lmarkzon** [comment]
@maya yes I will change the name of it and move it to styleguide.scss and push it up again!

**Comment by maya** [comment]
@lmarkzon thanks! Seems like there may be a merge conflict now, could you take a look? Once we get that resolved we'll be able to merge it in 🚀

**Pull request closed without merging**